src/Layouts/Main.purs

```diff
@@ -5,20 +5,19 @@ import Prelude
 import Bootstrap.Layout as L
 import Components.About as CAbout
 import Components.Home as CHome
-import Control.Monad.Aff (Aff, launchAff_)
-import Control.Monad.Eff (Eff)
 import Data.Either.Nested (Either4)
 import Data.Functor.Coproduct.Nested (Coproduct4)
 import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (Aff, launchAff_)
 import Halogen as H
-import Halogen.Aff as HA
 import Halogen.Component.ChildPath (cp1, cp2, cp3, cp4)
 import Halogen.HTML as HH
 import Halogen.HTML.Properties as HP
 import Layouts.Footer as LFooter
 import Layouts.Header as LHeader
 import Routers as RT
-import Routing (matches)
+import Routing (match)
```

Comment:
`matches` got deleted in the recent work. I think `match` is the new one. `matchWith` is the other one that is exposed, but I think it does not fit further down.